packages/frontend/src/components/NavBar/NotificationsMenu/index.tsx

```diff
@@ -1,28 +1,53 @@
+import { NotificationResourceType } from '@lightdash/common';
 import { Button, Indicator, Menu } from '@mantine/core';
 import { IconBell } from '@tabler/icons-react';
 import { FC } from 'react';
+import { useDashboardCommentsCheck } from '../../../features/comments';
+import {
+    DashboardCommentsNotifications,
+    useGetNotifications,
+} from '../../../features/notifications';
 import {
     useValidation,
     useValidationNotificationChecker,
     useValidationUserAbility,
 } from '../../../hooks/validation/useValidation';
+import { useApp } from '../../../providers/AppProvider';
 import MantineIcon from '../../common/MantineIcon';
 import { ValidationErrorNotification } from './ValidationErrorNotification';
 
 export const NotificationsMenu: FC<{ projectUuid: string }> = ({
     projectUuid,
 }) => {
+    const { user } = useApp();
+
+    // Validator notifications
     const { data: validationData } = useValidation(projectUuid, false);
     const canUserManageValidations = useValidationUserAbility(projectUuid);
     const [hasReadValidationNotification, setHasReadValidationNotification] =
         useValidationNotificationChecker();
-
     const hasValidationErrors = validationData && validationData?.length > 0;
 
+    // Dashboard comments notifications
+    const dashboardCommentsCheck = useDashboardCommentsCheck(user?.data);
+    const userCanViewDashboardComments =
+        !!dashboardCommentsCheck.isDashboardTileCommentsFeatureEnabled &&
```

Comment:
Consider enforcing these to be booleans (with a default to `false`) in `useDashboardCommentsCheck` to avoid the `!!` boolean cast.